### PR TITLE
added file not found exception

### PR DESCRIPTION
**Pull request by vivonk**
## Description

Added exception to deleting a file from local
## Related Issue

#458 

## How Has This Been Tested?

I have tested it locally by deleting some files manually then request of delete from webpage 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by agentmilindu** [comment]
@rehrumesh Let's merge this.

File: components/core/DiskMan.py [hunk 1]
```diff
@@ -24,5 +24,8 @@ def remove_files(days, rate):
     if verbose:
         print("files to be removed", files)
     for file in files:
-        os.unlink(file[0])
+        try:
+            os.unlink(file[0])
+        except FileNotFoundError:
+            print('following file was not found on his path, removing match from database', file[0])
```
Comment (agentmilindu) [inline components/core/DiskMan.py]: Should be  
> on **this** path

Shall we go with capitalized log messages, we can change other places later on. 
> **F**ollowing file was not found on his path, removing the match from the database

Comment (vivonk) [reply components/core/DiskMan.py]: capitalization makes bit weird to read I think

Comment (vivonk) [reply components/core/DiskMan.py]: changes have done

**Pull request merged**